src/main/java/net/springboot/ipotrading/FinancialProductController.java

```diff
@@ -2,31 +2,45 @@
 
 import net.springboot.ipotrading.model.FinancialProduct;
 import net.springboot.ipotrading.model.PrimeResponse;
+import net.springboot.ipotrading.model.Transaction;
 import net.springboot.ipotrading.service.FinancialProductService;
 import net.springboot.ipotrading.service.IpoService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
please do not go by * import, it includes unnecessary files,only import the required annotation